sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/AvailabilityStrategy.java

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import java.util.List;
+
+/**
+ * The type Availability strategy.
+ */
+public abstract class AvailabilityStrategy {
+    /**
+     * The Effective retry regions.
+     */
+    /**
+     * The Number of regions to try.
+     */
+    int numberOfRegionsToTry;
```

Comment:
this should be protected.

Reply:
Removed as this is not necessary anymore